scene/resources/importer_mesh.h

```diff
@@ -112,7 +112,9 @@ class ImporterMesh : public Resource {
 
 	void set_surface_material(int p_surface, const Ref<Material> &p_material);
 
-	void generate_lods(float p_normal_merge_angle, float p_normal_split_angle, Array p_skin_pose_transform_array);
+	void generate_lods(float p_normal_merge_angle, float p_normal_split_angle, float p_normal_weight, Array p_skin_pose_transform_array);
+
+	void _generate_lods_script(float p_normal_merge_angle, float p_normal_split_angle, Array p_skin_pose_transform_array);
```

Comment:
This should be moved to the private area and called:
```suggestion
	void _generate_lods_bind(float p_normal_merge_angle, float p_normal_split_angle, Array p_skin_pose_transform_array);
```

Reply:
Moved & renamed